plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -233,20 +269,24 @@ var JobListWidget = View.extend({
         }
         job.set(event.data);
         this._renderData();
-        this._highlightRecordIfOnList(event.data._id);
+        this.trySetHighlightRecord(event.data._id);
     },
 
     _jobCreated: function (event) {
         this._fetchWithFilter()
             .done(() => {
-                this._highlightRecordIfOnList(event.data._id);
+                this.trySetHighlightRecord(event.data._id);
             });
     },
 
-    _highlightRecordIfOnList: function (jobId) {
-        if (this.currentView === 'list') {
-            var tr = this.$('tr[g-job-id=' + jobId + ']').addClass('g-highlight');
-            setTimeout(() => tr.removeClass('g-highlight'), 1000);
+    trySetHighlightRecord: function (jobId) {
+        if (this.collection.find((job) => job.id === jobId)) {
```

Comment:
[`Collection.get`](http://backbonejs.org/#Collection-get) is the correct (and more efficient) way to look up a Model by id:
```
if (this.collection.get(jobId)) {
```